modules/sdk-lib-mpc/src/tss/ecdsa-dkls/util.ts

```diff
@@ -265,3 +274,77 @@ export async function generate2of2KeyShares(
   });
   return [partyA, partyB];
 }
+
+export function isGG18SigningMaterial(keyShare: string, walletPassphrase: string | undefined): boolean {
+  const prv = sjcl.decrypt(walletPassphrase, keyShare);
+  try {
+    const signingMaterial = JSON.parse(prv);
+    return (
+      signingMaterial.pShare &&
+      signingMaterial.bitgoNShare &&
+      (signingMaterial.userNShare || signingMaterial.backupNShare)
+    );
+  } catch (error) {
+    return false;
+  }
+}
+
+export async function getMpcV2RecoveryKeyShares(
```

Comment:
Add a jsdoc here.

Reply:
Also why do we need this function? Why can we not use `getUserAndBackupSession`. 

This lib is for low level primitives, bitgo abstractions should live elsewhere (sdk-core for example).